PyInstaller/hooks/hook-PySide2.py

```diff
@@ -10,13 +10,19 @@
 
 from PyInstaller.utils.hooks import collect_system_data_files
 from PyInstaller.utils.hooks.qt import pyside2_library_info, get_qt_binaries
+import PyInstaller.compat as compat
 
 hiddenimports = ['shiboken2']
 
 # Collect the ``qt.conf`` file.
+if compat.is_win:
+    target_qt_conf_dir = '.'
```

Comment:
For maximum compatibility, consider using `os.curdir`

Reply:
changed